CMakeLists.txt

```diff
@@ -129,6 +129,7 @@ option (WITH_PYTHON2_BINDINGS               "Build Python bindings for Python 2"
 option (WITH_PYTHON3_BINDINGS               "Build Python bindings for Python 3"                        OFF)
 option (WITH_DISNEY_MATERIAL                "Build Disney material"                                     OFF)
 option (WITH_EMBREE                         "Include support for Embree intersection backend"           OFF)
+option (WITH_SPECTRAL_COLOR                 "Include support for spectral colors"                       ON)
```

Comment:
I find `APPLESEED_WITH_SPECTRAL_COLOR` somewhat confusing since the pipeline still supports RGB colors when defined. `APPLESEED_WITH_SPECTRAL_SUPPORT` would be clearer and would help avoid the confusion that is certain to happen.